niworkflows/interfaces/cifti.py

```diff
@@ -161,9 +161,9 @@ def _create_cifti_image(bold_file, label_file, annotation_files, gii_files,
         label_img = nb.load(label_file)
         bold_img = resample_to_img(bold_file, label_img)
 
-        bold_data = bold_img.get_data()
+        bold_data = bold_img.get_fdata(dtype='float32')
         timepoints = bold_img.shape[3]
-        label_data = label_img.get_data()
+        label_data = np.asanyarray(label_img._dataobj, dtype='int16')
```

Comment:
Is this the proper way of accessing an int16 array?

Reply:
This may fail due to nipy/nibabel#843. I'd suggest an astype for now. Should be pretty cheap for a single volume, even if it does coerce up first.

Reply:
Just to be clear, you are suggesting:

```
label_img.dataobj.astype('int16')
```
?

Reply:
np.asanyarray(dataobj).astype('uint16')